PencilCase/PencilCase/scripts/CustomBindings/SelectComponentInitialValueBinding.ts

```diff
@@ -0,0 +1,12 @@
+﻿class SelectComponentInitialValueBinding implements KnockoutBindingHandler {
```

Comment:
Default select component UI doesn't display the selected item text (the textbox), so need this custom binding handler to do that.